src/amqp/connection.js

```diff
@@ -53,6 +54,9 @@ var Adapter = function( parameters ) {
 	var pfxPath = getOption( parameters, 'RABBIT_PFX' ) || getOption( parameters, 'pfxPath' );
 	var useSSL = certPath || keyPath || passphrase || caPaths || pfxPath;
 	this.options = { noDelay: true };
+  if (timeout){
```

Comment:
@derickbailey - this project uses tabs, can you please change 55-57 to follow that formatting?